recipe/meta.yaml

```diff
@@ -24,21 +22,20 @@ requirements:
     - wheel
   run:
     - python
-    - packaging >=17.1
-    - numpy >=1.17.2
-    - pyyaml >=5.4
-    - pytorch >=1.11.0
-    - tqdm >=4.57.0
-    - fsspec >2021.06.1
-    - torchmetrics >=0.7.0
-    - typing-extensions >=4.0.0
-    - lightning-utilities >=0.7.0
+    - packaging >=20.0,<=23.1
```

Comment:
you don't need upper bounds, see notes in requirements/pytorch/base.txt

Reply:
The Source Distribution on PyPI is missing the tests folder, see https://pypi.org/project/pytorch-lightning/#files
I tried to test it from the upstream github repo but the tests are quite heavy.

Another issue is that the Source Distribution on PyPI installs lightning_fabric and pytorch_lightning but if you install from github upstream tarballs then lightning will be installed but pytorch_lightning won't be available because of another structure. That will break the user experience.
I would follow the conda-forge approach using the PyPI source

Reply:
ah, ok. well, it's a pure python package isn't it. fine with me then